tests/Jenkins/dirac-cfg-update-services.py

```diff
@@ -48,38 +51,43 @@
 
 csAPI = CSAPI()
 
+fc = "FileCatalog"
+multiFC = "MultiVOFileCatalog"
+
+if os.environ["TEST_HTTPS"] == "Yes":
+    fc = f"Tornado{fc}"
+    multiFC = f"Tornado{multiFC}"
+
 for sct in [
     "Systems/DataManagement/Production/Services",
-    "Systems/DataManagement/Production/Services/FileCatalog",
-    "Systems/DataManagement/Production/Services/MultiVOFileCatalog",
+    f"Systems/DataManagement/Production/Services/{fc}",
+    f"Systems/DataManagement/Production/Services/{multiFC}",
 ]:
     res = csAPI.createSection(sct)
     if not res["OK"]:
         print(res["Message"])
-        exit(1)
+        sys.exit(1)
 
-csAPI.setOption("Systems/DataManagement/Production/Services/FileCatalog/DirectoryManager", "DirectoryClosure")
-csAPI.setOption("Systems/DataManagement/Production/Services/FileCatalog/FileManager", "FileManagerPs")
-csAPI.setOption("Systems/DataManagement/Production/Services/FileCatalog/SecurityManager", "VOMSSecurityManager")
-csAPI.setOption("Systems/DataManagement/Production/Services/FileCatalog/UniqueGUID", True)
+csAPI.setOption(f"Systems/DataManagement/Production/Services/{fc}/DirectoryManager", "DirectoryClosure")
+csAPI.setOption(f"Systems/DataManagement/Production/Services/{fc}/FileManager", "FileManagerPs")
+csAPI.setOption(f"Systems/DataManagement/Production/Services/{fc}/SecurityManager", "VOMSSecurityManager")
+csAPI.setOption(f"Systems/DataManagement/Production/Services/{fc}/UniqueGUID", True)
 
-csAPI.setOption("Systems/DataManagement/Production/Services/MultiVOFileCatalog/DirectoryManager", "DirectoryClosure")
-csAPI.setOption("Systems/DataManagement/Production/Services/MultiVOFileCatalog/FileManager", "FileManagerPs")
-csAPI.setOption("Systems/DataManagement/Production/Services/MultiVOFileCatalog/SecurityManager", "NoSecurityManager")
-csAPI.setOption("Systems/DataManagement/Production/Services/MultiVOFileCatalog/UniqueGUID", True)
+csAPI.setOption(f"Systems/DataManagement/Production/Services/{multiFC}/DirectoryManager", "DirectoryClosure")
```

Comment:
@atsareg @fstagni 
the options are set here, so sometimes with Tornado, sometimes just MultiVOFileCatalog

Reply:
Well, why then they are not set when the client test is run ? This has nothing to do with the service itself, pure CS

Reply:
By the way, this is the ONLY place in the integration tests where the presence of an option on the CS is checked. In other cases there are default values used in case of no CS. May be, this was never actually checked and gConfig does not serve the CS values at all ?

Reply:
They are set in a different place!?!